digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/ProjectSectionCalculator.scala

```diff
@@ -230,6 +282,7 @@ object ProjectSectionCalculator {
               assignCalibrationPoints(Seq(), sec.left.links, calMap)
           }
         }
+        //validateCalibrationPointsDiscontinuityOnTracks()
```

Comment:
Should this be removed or taken in use?

Reply:
I second that question.

Reply:
Well this method is not used at the moment because we didn't decide with Janne how it should behave. We should remove the usage and then add some comment to method, so we can reuse it when we have a more clear idea of want we need to do there.

Reply:
I will handle it.

Reply:
All right, thank you :)